src/isAnyTrust.ts

```diff
@@ -18,12 +26,24 @@ export async function isAnyTrust<TChain extends Chain | undefined>({
   const transaction = await publicClient.getTransaction({
     hash: createRollupTransactionHash,
   });
-  const {
-    args: [{ config }],
-  } = decodeFunctionData({
-    abi: [createRollupABI],
-    data: transaction.input,
-  });
-  const chainConfig = JSON.parse(config.chainConfig);
-  return chainConfig.arbitrum.DataAvailabilityCommittee;
+
+  try {
+    // try parsing for RollupCreator v2.1
+    const {
+      args: [{ config }],
+    } = decodeFunctionData({
+      abi: [createRollupABI],
+      data: transaction.input,
+    });
+    return parseConfig(config);
+  } catch (error) {
+    // try parsing for RollupCreator v1.1
+    const {
+      args: [{ config }],
+    } = decodeFunctionData({
+      abi: [createRollupV1Dot1ABI],
+      data: transaction.input,
+    });
+    return parseConfig(config);
+  }
```

Comment:
this works but looks pretty ugly

alternatively we can loop through an array of ABI to try parsing?